tests/sub_port_interfaces/test_sub_port_interfaces.py

```diff
@@ -66,3 +73,150 @@ def test_packet_routed_with_invalid_vlan(self, duthost, ptfadapter, apply_config
+            shutdown_port(duthost, sub_port)
+            generate_and_verify_traffic(duthost=duthost,
+                                        ptfadapter=ptfadapter,
+                                        src_port=value['neighbor_port'],
+                                        ip_src=value['neighbor_ip'],
+                                        dst_port=sub_port,
+                                        ip_dst=value['ip'],
+                                        pkt_action='drop')
+
+            for next_sub_port, next_value in sub_ports.items():
+                if next_sub_port != sub_port:
+                    generate_and_verify_traffic(duthost=duthost,
+                                                ptfadapter=ptfadapter,
+                                                src_port=next_value['neighbor_port'],
+                                                ip_src=next_value['neighbor_ip'],
+                                                dst_port=next_sub_port,
+                                                ip_dst=next_value['ip'],
+                                                pkt_action='fwd')
+
+            startup_port(duthost, sub_port)
+            generate_and_verify_traffic(duthost=duthost,
+                                        ptfadapter=ptfadapter,
+                                        src_port=value['neighbor_port'],
+                                        ip_src=value['neighbor_ip'],
+                                        dst_port=sub_port,
+                                        ip_dst=value['ip'],
+                                        pkt_action='fwd')
+
+
+    def test_max_numbers_of_sub_ports(self, duthost, ptfadapter, apply_config_on_the_dut, apply_config_on_the_ptf):
+        """
+        Validates that 256 sub-ports can be created per port or LAG
+
+        Test steps:
+            1.) Setup configuration of 256 sub-ports on the DUT.
+            2.) Setup configuration of 256 sub-ports on the PTF.
+            3.) Create ICMP packet.
+            4.) Send ICMP request packet from PTF to DUT.
+            5.) Verify that DUT sends ICMP reply packet to PTF.
+            6.) Clear configuration of sub-ports on the DUT.
+            7.) Clear configuration of sub-ports on the PTF.
+
+        Pass Criteria: PTF gets ICMP reply packet from DUT.
+
+        Note:
+            The running of the test case takes about 80 minutes.
+        """
+        sub_ports = apply_config_on_the_dut['sub_ports']
+
+        for sub_port, value in sub_ports.items():
+            generate_and_verify_traffic(duthost=duthost,
+                                        ptfadapter=ptfadapter,
+                                        src_port=value['neighbor_port'],
+                                        ip_src=value['neighbor_ip'],
+                                        dst_port=sub_port,
+                                        ip_dst=value['ip'],
+                                        pkt_action='fwd')
```

Comment:
This test case needs too much time (80 minutes) to run. Can you simplify it a little bit:
* Verify that 256 sub-ports are created.
* Verify traffic on the first, last sub-port and a random one in the middle.

Then it would not need that much time to run it.